libs/ledger-live-common/src/families/avalanchepchain/hw-app-avalanche.ts

```diff
@@ -0,0 +1,303 @@
+import Transport from "@ledgerhq/hw-transport";
+import BIPPath from "bip32-path";
+import bech32 from "bech32";
+import secp256k1 from "secp256k1";
+import { createHash } from "crypto";
+import { AVAX_HRP } from "./utils";
+
+const CLA = 0x80;
+const INS = {
+  GET_VERSION: 0x00,
+  GET_ADDRESS: 0x02,
+  GET_EXTENDED_PUBLIC_KEY: 0x03,
+  SIGN_HASH: 0x04,
+  SIGN_TRANSACTION: 0x05,
+};
+const MAX_APDU_SIZE = 230;
+
+export const AVAX_BIP32_PREFIX = "m/44'/9000'/0'";
+
+export default class Avalanche {
```

Comment:
Hey, sorry! I was not aware that there was a preferred architecture for these packages. Way back when we were using `yalc`, writing this code inside of `families` ensured there was less chance for error. 

I'd be happy to change this if necessary, though maybe in a follow-up PR since this one is getting pretty massive?

Reply:
I've removed this file completely and am instead relying on the [zondax](https://github.com/ava-labs/ledger-avalanche/tree/main/js) app.